components/badge/badge.css

```diff
@@ -0,0 +1,60 @@
+.badge-container{
```

Comment:
Naming convention is super simple and understandable.

Reply:
Thank you for the review